ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/toolAdmin/ToolAdminViewDefaultAppearance.java

```diff
@@ -0,0 +1,412 @@
+        return displayStrings.containerVolumesFromTagLabel();
+    }
+
+    @Override
+    public int containerVolumesFromTagWidth() {
+        return 50;
+    }
+
+    @Override
+    public String containerVolumesFromURLLabel() {
+        return displayStrings.containerVolumesFromURLLabel();
+    }
+
+    @Override
+    public int containervolumesFromURLWidth() {
+        return 350;
+    }
+
+    @Override
+    public String containerVolumesFromNamePrefixLabel() {
+        return displayStrings.containerVolumesFromNamePrefixLabel();
+    }
+
+    @Override
+    public int containerVolumesFromNamePrefixWidth() {
+        return 150;
+    }
+
+    @Override
+    public String containerVolumesFromReadOnlyLabel() {
+        return displayStrings.containerVolumesFromReadyOnlyLabel();
+    }
+
+    @Override
+    public int containerVolumesFromReadOnlyWidth() {
+        return 75;
+    }
+
+    @Override
+    public String toolImplementationFieldSetLabel() {
+        return displayStrings.toolImplementationFieldSetLabel();
+    }
+
+    @Override
+    public SafeHtml toolImplementationImplementorLabel() {
+        return templates.requiredFieldLabel(displayStrings.toolImplementationImplementorLabel());
+    }
+
+    @Override
+    public SafeHtml toolImplementationImplementorEmailLabel() {
+        return templates.requiredFieldLabel(displayStrings.toolImplementationImplementorEmailLabel());
+    }
+
+    @Override
+    public String toolTestDataParamsLabel() {
+        return displayStrings.testToolDataParamsLabel();
+    }
+
+    @Override
+    public int toolTestDataParamsWidth() {
+        return 500;
+    }
+
+    @Override
+    public String toolTestDataInputFilesLabel() {
+        return displayStrings.testToolDataInputFilesLabel();
+    }
+
+    @Override
+    public int toolTestDataInputFilesWidth() {
+        return 500;
+    }
+
+    @Override
+    public String toolTestDataOutputFilesLabel() {
+        return displayStrings.testToolOutputFilesLabel();
+    }
+
+    @Override
+    public int toolTestDataOutputFilesWidth() {
+        return 500;
+    }
+
+    @Override
+    public String dialogWindowName() {
+        return displayStrings.dialogWindowName();
+    }
+
+    @Override
+    public String dialogWindowUpdateBtnText() {
+        return displayStrings.dialogWindowUpdateBtnText();
+    }
+
+    @Override
+    public String addToolSuccessText() {
+        return displayStrings.addToolSuccessText();
+    }
+
+    @Override
+    public String updateToolSuccessText() {
+        return displayStrings.updateToolSuccessText();
+    }
+
+    @Override
+    public String dialogWindowDeleteBtnText() {
+        return displayStrings.dialogWindowDeleteBtnText();
+    }
+
+    @Override
+    public String deleteToolSuccessText() {
+        return displayStrings.deleteToolSuccessText();
+    }
+
+    @Override
+    public String publicSubmitError() {
+        return iplantDisplayStrings.publicSubmitTip();
+    }
```

Comment:
This error message is used in a few places, but I think its name a little confusing.
It's a good idea to reuse this `Please complete required fields` error message, but since it's now being used outside of the `Make app public` context, this property/method name should now be renamed to something more generic. Maybe `completeRequiredFieldsError`.